base/jobs/docker/1.0/py3/Dockerfile.cpu

```diff
@@ -121,8 +121,9 @@ RUN ${PIP} install --no-cache --upgrade \
         PennyLane-Lightning==0.31.0 \
         scikit-learn==1.2.2 \
         requests==2.26.0 \
-        scipy==1.9.3
-
+        scipy==1.9.3 \
+        qiskit-braket-provider==0.0.4
```

Comment:
Weird. Why is this different than what we [have in the NBIs](https://github.com/amazon-braket/amazon-braket-examples/blob/main/requirements.txt#L24C1-L24C23)?

Reply:
Since Pypi uses qiskit-braket-provider: https://pypi.org/project/qiskit-braket-provider/, I will keep this format and publish a PR to update the example repo: https://github.com/amazon-braket/amazon-braket-examples/pull/404